components/docusign/actions/create-draft/create-draft.mjs

```diff
@@ -0,0 +1,16 @@
+import createSignatureRequest from "../create-signature-request/create-signature-request.mjs";
+
+export default {
+  ...createSignatureRequest,
+  key: "docusign-create-draft",
+  version: "0.0.1",
+  name: "Create Draft",
+  description: "Creates and sends an envelope or creates a draft envelope. [See the docs here](https://developers.docusign.com/docs/esign-rest-api/reference/envelopes/envelopes/create/)",
```

Comment:
```suggestion
  description: "Create and send an envelope or create a draft envelope. [See the documentation here](https://developers.docusign.com/docs/esign-rest-api/reference/envelopes/envelopes/create/)"
```

Reply:
Changed 'Creates' to 'Create', 'sends' to 'send', and 'docs' to 'documentation' to follow Google's Developer docs style guide (use present tense and prefer 'documentation' over 'docs')